### PR TITLE
Add initial CI configuration

### PR DESCRIPTION
**Pull request by keith**

### CODE REVIEW

**Comment by keith** [comment]
https://github.com/bazelbuild/continuous-integration/pull/1425

File: .bazelci/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+buildifier: latest
+platforms:
+  ubuntu1804:
+    working_directory: examples/basic
+    build_targets:
```
Comment (ahumesky) [inline .bazelci/presubmit.yml:6]: i think `--fat_apk_cpu=arm64-v8a,x86` and `--android_crosstool_top=@androidndk//:toolchain` will also be needed, which i think can be added like this:
https://github.com/bazelbuild/bazel/blob/d3bef49f9182f6bcb6f609530970a7945ce51cbf/.bazelci/presubmit.yml#L117

(and for `macos` below)

**Review by ahumesky — CHANGES_REQUESTED** [review]

File: .bazelci/presubmit.yml (hunk 1 above)
Comment (keith) [reply .bazelci/presubmit.yml:6]: oops yep, sorry

**Review by ahumesky — APPROVED** [review]

**Comment by ahumesky** [comment]
I believe that this and https://github.com/bazelbuild/continuous-integration/pull/1425 will add presubmit tests for changes to bazel itself, but not for changes to this repo (or at minimum it might require additional setup)

I filed https://github.com/bazelbuild/rules_android_ndk/issues/12
